charter.md

```diff
@@ -0,0 +1,336 @@
+# observe-k8s working document
+
+**Table of Contents**
```

Comment:
I think the ToC is way too detailed. Let's prune it so that the important bits are not buried.

Reply:
agreed, for now nuking it entirely.